<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_PK_Dialog</class>
 <widget class="QDialog" name="add_PK_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新增封装订单</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>新增物资信息：</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1,2,1,2,1,2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>晶圆批号：</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="waferNum_comboBox">
          <item>
           <property name="text">
            <string>001</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>002</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>CP测试单号：</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cpTestNum_comboBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>PK封装单号：</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="package_lineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>封装形式：</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="packageType_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,1,2,1,2,1,2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>封装规范：</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="packageSpecification_lineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>封装投入：</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="packageInvestment_lineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>特殊加工需求：</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="process_demand_lineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>封装产出：</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="package_output_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>备注：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="note_lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="ok_pushButton">
          <property name="text">
           <string>确定</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>image/update.png</normaloff>image/update.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>晶圆批号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CP测试单号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PK封装单号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>封装形式</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>封装规范</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>封装投入</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>特殊加工需求</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>封装产出</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>备注</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addPK_pushButton">
       <property name="text">
        <string>添加</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>image/move.png</normaloff>image/move.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
